### ISSUE TITLE
Texturing mapping error on Service Module Adapter

### ISSUE THREAD

**Issue by sumghai** [sumghai/SDHI_ServiceModuleSystem#44]
Forum user **Bomoo** has discovered a slight misalignment in the UV mapping for the Service Module Adapter texture:

![220200_screenshots_2015-05-24_00001](https://cloud.githubusercontent.com/assets/4993380/7791710/9c062e4c-02f9-11e5-9e7b-058c96c11bd1.jpg)

Should be an easy fix.

**Comment by sumghai** [comment]
UV maps and textures have been updated (5998d92), awaiting **Bomoo**'s feedback.

**Comment by sumghai** [comment]
Based on **Bomoo**'s feedback and problems encountered with the edges of the UV island making up the adapter base ring, I ended up revising the texture so that there are six panel lines instead of the original two, and reoriented them so that they don't coincide with the aforementioned troublesome UV edges (ef9837d). 

As an added bonus, the new adapter panel lines happen to coincide with the similarly six-segmented Service Module upper ring, which is a nice visual continuity.

More feedback from **Bomoo** later, I tweaked the normal maps to mitigate some strange bright spots that were otherwise only present when subjected to direct close-range illumination (ea95ee1).

This will have to do.
